README.md

```diff
@@ -21,20 +23,57 @@ In your `build.gradle` file, add this line:
 ```groovy
 dependencies {
     // ...
-    api 'com.pusher:pusher-platform-android:0.3.1'
+    api 'com.pusher:pusher-platform-android:$pusher_platform_version'
 }
 ```
 
-## Testing
+## Usage
+
+The main entity used to access to platform services is [Instance](pusher-platform-core/src/main/kotlin/com/pusher/platform/Instance.kt).
+
+The simpler way to create a new instance uses 4 things:
 
-### TODO
+* locator: Instance locator id in the form of 'v1:us1:1a234-123a-1234-12a3-1234123aa12'
```

Comment:
Nitpick: I'd put locator, serviceName etc in backticks to get `locator`, `serviceName` etc